diff --git a/node_modules/cz-customizable/lib/read-config-file.js b/node_modules/cz-customizable/lib/read-config-file.js
index 45e4c0e..32dcf74 100755
--- a/node_modules/cz-customizable/lib/read-config-file.js
+++ b/node_modules/cz-customizable/lib/read-config-file.js
@@ -4,7 +4,7 @@ const findConfig = require('find-config');
 const path = require('path');
 const log = require('./logger');
 
-const readConfigFile = (CZ_CONFIG_NAME = '.cz-config.js') => {
+const readConfigFile = (CZ_CONFIG_NAME = '.cz-config.cjs') => {
   // First try to find the .cz-config.js config file
   // It seems like find-config still locates config files in the home directory despite of the home:false prop.
   const czConfig = findConfig.require(CZ_CONFIG_NAME, { home: false });
